helmctlr/helm_test.go

```diff
@@ -124,6 +143,10 @@ func TestResourceAddedHappyPath(t *testing.T) {
 	defer mockCtrl.Finish()
 	mockHelm := NewMockInterface(mockCtrl)
 	testController.Helm = mockHelm
+
+	k8sClient := fake.NewSimpleClientset(testHelmReleaseConfigmap)
+	testController.KubeClient = k8sClient
```

Comment:
I would rather see testController take the kubernetes client as an input instead of making KubeClient public on the controller type

Reply:
Ah yes, I wasn't able to do that before cause I was using a generated mock rather than the fake client. Part of this all is because the current tests use a global testController.

Reply:
Fixed.